art/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Would you please revert this default encoding?
We need this one for some special cases.